shopfloor/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 {
     "name": "Shopfloor",
     "summary": "manage warehouse operations with barcode scanners",
-    "version": "14.0.4.2.4",
+    "version": "14.0.5.0.0",
```

Comment:
Maybe this version is enough
```suggestion
    "version": "14.0.4.3.0",
```